ext/civicrm_admin_ui/managed/SavedSearch_SMS_Provider.mgd.php

```diff
@@ -0,0 +1,181 @@
+<?php
+use CRM_CivicrmAdminUi_ExtensionUtil as E;
+
+return [
+  [
+    'name' => 'SavedSearch_SMS_Provider',
+    'entity' => 'SavedSearch',
+    'cleanup' => 'unused',
+    'update' => 'unmodified',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'name' => 'SMS_Provider',
+        'label' => E::ts('SMS Provider'),
+        'api_entity' => 'Provider',
```

Comment:
Per https://github.com/civicrm/civicrm-core/pull/29555 this will need to be changed to 

```suggestion
        'api_entity' => 'SmsProvider',
```